hack/images/kubeone-e2e/install-kube-tests-binaries.sh

```diff
@@ -22,17 +22,29 @@ full_versions["1.17"]="v1.17.4"
 full_versions["1.18"]="v1.18.0"
 
 root_dir=${KUBETESTS_ROOT:-"/opt/kube-test"}
+tmp_root=${TMP_ROOT:-"/tmp/get-kube"}
 
 for version in "${!full_versions[@]}"; do
     full_version="${full_versions[${version}]}"
     directory="${root_dir}/kubernetes-${version}"
+    tmp_dir="${tmp_root}/kubernetes-${version}"
     if [[ ! -d "${directory}" ]]; then
+        mkdir -p "${tmp_dir}"
         mkdir -p "${directory}"
-        cd "${directory}"
-        kubetest --extract="${full_version}"
```

Comment:
As said in the PR description, some of kubetest scripts now require gcloud, which we don’t have in our image. To avoid installing and configuring it, I decided to try this approach.